src/constants.ts

```diff
@@ -13,21 +13,57 @@ import qualityAssuranceQuiz from "./data/quality-assurance-quiz";
 import securityQuiz from "./data/security-quiz";
 import sqlQuiz from "./data/sql-quiz";
 
+export interface QuizQuestion {
+  Question: string;
+  Answer: string;
+  Distractor1: string;
+  Distractor2: string;
+  Distractor3: string;
+  Explanation: string;
+  Link?: string;
+  Category?: string;
+}
+
+export const CATEGORY_NAMES = {
+  ACCESSIBILITY: "Accessibility",
+  AGILE: "Agile",
+  CSS: "CSS",
+  FREECODECAMP: "freecodecamp",
+  GENERAL: "General Computer Science",
+  GIT: "Git",
+  HTML: "Html",
+  JAVASCRIPT: "Javascript",
+  INFOTECH: "Infomation Technology",
+  LINUX: "Linux",
+  PYTHON: "Python",
+  SECURITY: "Security",
+  SQL: "SQL",
+  QUALITYASSURANCE: "Quality Assurance"
+};
+
+function addCategoryToQuiz(quizQuestion: QuizQuestion[], category: string) {
+  const questionHolder: QuizQuestion[] = [];
+  quizQuestion.map(x =>
+    questionHolder.push(Object.assign({}, x, { Category: category }))
+  );
+  return questionHolder;
+}
```

Comment:
This function was designed to add the category name itself into the question object that way I can keep track of score for the categories. This seemed like the best way to go about getting the data in the right place.

Reply:
Ok. Would it be possible to use a `forEach` here instead of map?
Because it doesn't look like you are using the new array that map gives so maybe `forEach`  would work better.

Reply:
That works.